packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/AutoPauseButton.tsx

```diff
@@ -0,0 +1,143 @@
+import { selectIsSessionAutopaused } from '@wallet-reducers/coinjoinReducer';
+import { useDispatch } from '@suite-hooks/useDispatch';
+
+const TRANSITION_CONFIG = '0.1s ease';
+
+const StyledIcon = styled(Icon)<{ isActivated: boolean }>`
+    position: absolute;
+    left: 6px;
+    padding: 6px;
+    border: 1.5px solid
+        ${({ theme, isActivated }) => (isActivated ? theme.TYPE_GREEN : theme.TYPE_LIGHT_GREY)};
+    border-radius: 50%;
+    transition: background ${TRANSITION_CONFIG}, border-color ${TRANSITION_CONFIG};
+
+    path {
+        transition: fill ${TRANSITION_CONFIG};
+    }
+`;
+
+const getHoverStyle = (backgroundColor: string, strokeColor: string) => css`
+    color: ${strokeColor};
+    background: ${backgroundColor};
+    border-color: ${backgroundColor};
+
+    ${StyledIcon} {
+        background: ${strokeColor};
+        border-color: ${strokeColor};
+
+        path {
+            fill: ${backgroundColor};
+        }
+    }
+`;
+
+const Container = styled.button<{
+    isActivated: boolean;
+    isHovered: boolean;
+    initialWidth: number | undefined;
+}>`
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: ${({ initialWidth }) => `${initialWidth}px` || 'fit-content'};
+    margin: 14px auto 0;
+    padding: 3px 8px 3px 26px;
+    border: 1px solid ${({ theme }) => theme.STROKE_LIGHT_GREY};
+    border-radius: 20px;
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    background: none;
+    transition: background ${TRANSITION_CONFIG}, border-color ${TRANSITION_CONFIG},
+        color ${TRANSITION_CONFIG};
+    cursor: pointer;
+    appearance: none;
+
+    :focus-visible {
+        ${({ theme }) => getHoverStyle(theme.BG_GREY, theme.TYPE_LIGHTER_GREY)}
+    }
+
+    ${({ theme, isHovered }) => isHovered && getHoverStyle(theme.BG_GREY, theme.TYPE_LIGHTER_GREY)}
+
+    ${({ theme, isActivated, isHovered }) =>
+        isActivated &&
+        css`
+            border-color: ${theme.BG_LIGHT_GREEN};
+            background: ${theme.BG_LIGHT_GREEN};
+            color: ${theme.TYPE_GREEN};
+
+            :focus-visible {
+                ${getHoverStyle(theme.BG_LIGHT_RED, theme.TYPE_RED)}
+            }
+
+            ${isHovered && getHoverStyle(theme.BG_LIGHT_RED, theme.TYPE_RED)}
+        `};
+`;
+
+interface AutoPauseButtonProps {
+    relatedAccountKey: string;
+}
+
+export const AutoPauseButton = ({ relatedAccountKey }: AutoPauseButtonProps) => {
+    const isActivated = useSelector(state => selectIsSessionAutopaused(state, relatedAccountKey));
+    const [initialWidth, setInitialWidth] = useState<number | undefined>();
+    const [isHovered, setIsHovered] = useState(false);
+
+    const theme = useTheme();
+    const dispatch = useDispatch();
+    const [wrapperRef, { width }] = useMeasure<HTMLButtonElement>();
+
+    useEffect(() => {
```

Comment:
not needed anymore since the copy changed and it would require even more logic to adapt to the longest copy, which is not the one rendered initially :)